### PR TITLE
probe launcher merge

### PR DESCRIPTION
**Pull request by amazingalek**
merged from:

- dev
- sector-sync-fix (#115)
- closest-sector (#116)

### CODE REVIEW

File: QSB/Tools/PlayerToolsManager.cs [hunk 1]
```diff
@@ -27,8 +27,17 @@ public static void Init(Transform camera)
 
             CreateFlashlight();
             CreateSignalscope();
-            CreateTranslator();
-            CreateProbeLauncher();
```
Comment (misternebula) [inline QSB/Tools/PlayerToolsManager.cs:31]: ?

File: QSB/Tools/PlayerToolsManager.cs [hunk 2]
```diff
@@ -96,9 +105,9 @@ private static void CreateSignalscope()
             signalscopeRoot.SetActive(true);
         }
 
-        private static void CreateTranslator()
+        private static void CreateTranslator(NomaiTranslatorProp translatorProp)
         {
-            var translatorRoot = Object.Instantiate(GameObject.Find("NomaiTranslatorProp"));
+            var translatorRoot = GameObject.Instantiate(translatorProp.gameObject);
```
Comment (misternebula) [inline QSB/Tools/PlayerToolsManager.cs:111]: Object v GameObject? which should we do

Comment (amazingalek) [reply QSB/Tools/PlayerToolsManager.cs:111]: Object I guess

Comment (amazingalek) [reply QSB/Tools/PlayerToolsManager.cs]: fixed

**Pull request merged**